tf_agents/agents/ppo/ppo_agent.py

```diff
@@ -165,6 +166,9 @@ def __init__(self,
       reward_norm_clipping: Value above and below to clip normalized reward.
       normalize_observations: If true, keeps moving mean and variance of
         observations and normalizes incoming observations.
+      train_on_partial_episodes: If True (default False), then data from
```

Comment:
let's remove this arg and assume it's always True; per oscar's comment on original PR.